lib/matplotlib/text.py

```diff
@@ -455,6 +457,8 @@ def _get_layout(self, renderer):
                 offsety = ymax1
             elif valign == 'baseline':
                 offsety = ymax1 - baseline
+            elif valign == 'center_baseline':
+                offsety = (ymin1 + ymax1) / 2.0
```

Comment:
In other words, it is identical to 'center'.  Maybe it would be better to just make that explicit by using `elif valign in ('center', 'center_baseline')`.  Would a short comment be enough to explain why this is?